x-pack/platform/plugins/private/upgrade_assistant/server/lib/reindexing/reindex_service.ts

```diff
@@ -169,7 +169,12 @@ export const reindexServiceFactory = (
    * @param reindexOp
    */
   const setReadonly = async (reindexOp: ReindexSavedObject) => {
-    const { indexName } = reindexOp.attributes;
+    const { indexName, rollupJob } = reindexOp.attributes;
```

Comment:
Note: I don't see this function being used as part of making indices read only but I don't have a full understanding of this code so I thought it was safer to keep this in than to remove it.